l402/header.go

```diff
@@ -71,14 +72,22 @@ func FromHeader(header *http.Header) (*macaroon.Macaroon, lntypes.Preimage, erro
 		macBase64, preimageHex := matches[2], matches[3]
 		macBytes, err := base64.StdEncoding.DecodeString(macBase64)
 		if err != nil {
-			return nil, lntypes.Preimage{}, fmt.Errorf("base64 "+
-				"decode of macaroon failed: %v", err)
+			// macBase64 might be multiple macaroons separated by commas,
```

Comment:
nit: comments should always be full sentences, including punctuation.

I also think this needs some more explanation that we're using `macaroon.Slice` below, which can actually decode multiple macaroons that are in one continuous blob of data.